### ISSUE TITLE
Add color support

### ISSUE THREAD

**Issue by ivanakcheurov** [jmespath/jp#38]
Please could you add color support similar to `jq`?

**Comment by wryun** [comment]
Many years ago: https://github.com/jmespath/jp/pull/13

Binary available here: https://github.com/wryun/jp/releases/tag/0.2.0

**Comment by rebelnn** [comment]
I got 0.2.1 doesn't seem to have color support in it

```
~|⇒ jp --version
jp version 0.2.1

```

**Comment by wryun** [comment]
Yes, because that PR hasn't been merged in. You can either build it yourself (with that PR, or some version of it), or use the binary that I linked.

**Comment by cschar** [comment]
+1 for supporting color. this is the selected language for parsing `az cli` query stuff..
